NEWS.md

```diff
@@ -41,11 +41,13 @@ Standard library changes
 
 * `dot` now admits a 3-argument method `dot(x, A, y)` to compute generalized dot products `dot(x, A*y)`, but without computing and storing the intermediate result `A*y` ([#32739]).
 
-#### SparseArrays
+#### Random
 
+* `AbstractRNG`s now behave like scalars when used in broadcasting ([#31071, #33213]).
```

Comment:
This won't work.  The issue links have to be in the `[#nnnnnn]` format links to be generated.

I think that we normally just link to the PR in NEWS, so `[#33213]` should be sufficient here